testsuite/internal.ml

```diff
@@ -182,21 +184,21 @@ let () = test "CmxsStubLink"
            T.f "libfoo_stubs.clib" ~content:"foo_stubs.o";
            T.f "foo.ml" ~content:"";
          ];
-         T.f "_tags" ~content:"
-<src/foo.{cma,cmxa}> : record_foo_stubs
-<src/foo.cmxs> : link_foo_stubs";
-         T.f "myocamlbuild.ml" ~content:"
-open Ocamlbuild_plugin
-let () =
-  dispatch begin function
-  | After_rules ->
-      dep [\"record_foo_stubs\"] [\"src/libfoo_stubs.a\"];
-      flag_and_dep
-        [\"link\"; \"ocaml\"; \"link_foo_stubs\"] (P \"src/libfoo_stubs.a\");
-      flag [\"library\"; \"ocaml\";           \"record_foo_stubs\"]
-        (S ([A \"-cclib\"; A \"-lfoo_stubs\"]));
-  | _ -> ()
-  end
+         T.f "_tags" ~content:"\
+<src/foo.{cma,cmxa}> : record_foo_stubs\n\
+<src/foo.cmxs> : link_foo_stubs\n";
+         T.f "myocamlbuild.ml" ~content:"\
+open Ocamlbuild_plugin\n\
+let () =\n\
+  dispatch begin function\n\
+  | After_rules ->\n\
+      dep [\"record_foo_stubs\"] [\"src/libfoo_stubs.a\"];\n\
+      flag_and_dep\n\
+        [\"link\"; \"ocaml\"; \"link_foo_stubs\"] (P \"src/libfoo_stubs.a\");\n\
+      flag [\"library\"; \"ocaml\";           \"record_foo_stubs\"]\n\
+        (S ([A \"-cclib\"; A \"-lfoo_stubs\"]));\n\
+  | _ -> ()\n\
+  end\n\
```

Comment:
Urgh, can you use quoted literals instead to avoid the many escapes here? (If there is a problem with `\r\n` somehow, we could inject a small helper function that removes the `\r`.)

Reply:
I've pushed https://github.com/ocaml/ocamlbuild/pull/329/commits/0f2342a2236c60252b638617c9c11752a7f6189c